amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/util/ISO8601DateTimeSerializer.java

```diff
@@ -0,0 +1,40 @@
+package org.digijava.kernel.ampapi.endpoints.util;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import io.swagger.models.properties.DateProperty;
+import io.swagger.models.properties.Property;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import org.digijava.kernel.ampapi.endpoints.common.EPConstants;
+import org.digijava.kernel.ampapi.swagger.converters.PropertyDescriber;
+
+/**
+ * 
+ * @author Nadejda Mandrescu
+ */
+public class ISO8601DateTimeSerializer extends JsonSerializer<Date> implements PropertyDescriber {
```

Comment:
API serializing classes are badly organized, placed wherever you feel like suitable at the moment:
`ISO8601DateSerializer` is under `org.digijava.kernel.ampapi.endpoints.util`
while
`ISO8601TimeStampSerializer` is under `org.digijava.kernel.services.sync.model`
Therefore as a dev once you find a Date serializer under "ampapi", you may not think that someone would define the DateTime serializer in another place, especially outside "ampapi". You would diligently create the apparently missing "DateTime" serializer under its proper place, as I did. Obviously I wouldn't have create a duplicate if I knew the existence of `ISO8601TimeStampSerializer`, don't you think? I am going to fix the bad structuring by moving the ISO8601TimeStampSerializer under ampapi.

Reply:
Agree with the fact that the class should be replaced in another package. The important thing is to have just one class. Maybe to put all serializers in one package in order to avoid such situations in the future.

Reply:
As I already explained that I'll just move them together, I have already did - why don't you check for changes?